docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -67,6 +67,19 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 
 ## Installing custom checkouts  
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--note">
+<div class="HubBlock-content">
+  
+  <!-- theme: note -->
+> 1. Please get in touch with support if you wish to customize checkout. 
+> 2. Enter your CSS into the [optimized-checkout.scss](https://github.com/bigcommerce/cornerstone/blob/master/assets/scss/optimized-checkout.scss) in Cornerstone or the [scss folder](https://github.com/bigcommerce/checkout-js/tree/master/src/scss) in checkout.js. We suggest moving content from "optimized-checkout.scss to the scss folder or vice-versa to ensure there is a single source of truth for handling styles. NOTE: "optimized-checkout.scss" will always override the CSS rules defined in checkout-js.
```

Comment:
Up at the top, the article says that the JS loader file must be responsible for loading all assets, including CSS. Does this conflict with being able to enter CSS or SCSS into the `optimized-checkout.scss` file?